packages/storybook/stories/va-segmented-progress-bar-uswds.stories.jsx

```diff
@@ -26,8 +26,9 @@ const Template = ({
   centeredLabels,
   counters,
   headerLevel,
-  progressTerm
-
+  progressTerm,
+  'use-div': useDiv,
+  'describedby-ids': describedbyIds,
```

Comment:
If we're not needing aria-describedby, we can probably remove this too.

Reply:
Thanks for finding this, thought I'd gotten all of the instances!